app/serializers/rest/block_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class REST::BlockSerializer < ActiveModel::Serializer
+  attributes :stealth
+
+  belongs_to :target_account, serializer: REST::AccountSerializer
+
+  def stealth
+    ActiveModel::Type::Boolean.new.cast(object.stealth)
```

Comment:
Dealing with a null value should probably happen in the model itself instead of here, and a simple `.nil?` conditional should suffice instead of calling this method which handles strings like "f", "false" etc.

Reply:
I actually don't remember why I did that :/
`ActiveModel::Type::Boolean.new.cast` returns `nil` for `nil` arguments, but there's no reason to make that distinction, so calling `.stealth?` should be enough. Going to change that.